src/editor/Neovim/NeovimHelpers.re

```diff
@@ -0,0 +1,33 @@
+open Oni_Core.Utility;
+
+module M = Msgpck;
+
+let convertNeovimExtType = (buffer: M.t) =>
+  switch (buffer) {
+  | M.Ext(kind, id) => Some((kind, convertUTF8string(id)))
+  | _ => None
+  };
+
+let getAtomicCallsResponse = response =>
```

Comment:
split out a helper function for dealing with the result of an `atomic_call` this returns `(option(errorList), option(resultList)` so we dont have to figure out the nesting repeatedly which I've found myself doing *a lot*

Reply:
Cool! Saw this used in a few places. Good call on refactoring this 👍